### PR TITLE
Fix/ui node.js version doc update 2.6.0 

### PR DESCRIPTION
**Pull request by ajainarayanan**
Associated JIRA: https://issues.cask.co/browse/CDAP-1804

### CODE REVIEW

**Comment by awholegunch** [comment]
Another place this is needed is:

- ~/Source/cdap_2.6/cdap-docs/admin-manual/source/installation/installation.rst

**Comment by ajainarayanan** [comment]
@awholegunch Updated the PR with update to installation.rst doc.

File: cdap-docs/admin-manual/source/installation/installation.rst [hunk 1]
```diff
@@ -127,10 +127,10 @@ Once you have installed the JDK, you'll need to set the JAVA_HOME environment va
 
 Node.js Runtime
 +++++++++++++++
-You can download the latest version of Node.js from `nodejs.org <http://nodejs.org>`__:
+You can download the appropriate version of Node.js (from v0.8.16 through v0.10.32) from `nodejs.org <http://nodejs.org>`__:
 
-1. Download the appropriate Linux or Solaris binary ``.tar.gz`` from
-   `nodejs.org/download/ <http://nodejs.org/download/>`__.
+#. Download the appropriate Linux or Solaris binary ``.tar.gz`` for a version in the above range from
+    `nodejs.org/dist/ <http://nodejs.org/dist/>`__.
```
Comment (awholegunch) [inline cdap-docs/admin-manual/source/installation/installation.rst]: remove the leading space (just before the \` ) so that the \`  lines up with the "D"

**Comment by awholegunch** [comment]
One comment, fix and merge. OLGTM.

**Comment by awholegunch** [comment]
LGTM.

**Pull request merged**